assets/js/blocks/featured-category/block.js

```diff
@@ -313,47 +408,73 @@ const FeaturedCategory = ( {
 			dimRatioToClass( dimRatio ),
 			contentAlign !== 'center' && `has-${ contentAlign }-content`
 		);
-		const mediaSrc = attributes.mediaSrc || getCategoryImageSrc( category );
-		const style = !! category ? getBackgroundImageStyles( mediaSrc ) : {};
-		if ( focalPoint ) {
-			const bgPosX = focalPoint.x * 100;
-			const bgPosY = focalPoint.y * 100;
-			style.backgroundPosition = `${ bgPosX }% ${ bgPosY }%`;
-		}
 
-		const onResizeStop = ( event, direction, elt ) => {
-			setAttributes( { height: parseInt( elt.style.height, 10 ) } );
+		const containerStyle = {
+			borderRadius: style.border.radius,
```

Comment:
Block is erroring in the editor due to this.

```suggestion
			borderRadius: style?.border?.radius,
```